src/Uuid/Uuids.php

```diff
@@ -87,7 +92,7 @@ public static function each(Closure $callback, string $type = 'all'): void
 	public static function generate(string $type = 'all'): void
 	{
 		static::each(
-			fn (Identifiable $model) => Uuid::for($model)->id(),
+			fn (Identifiable $model) => Uuid::for($model)?->id(),
```

Comment:
This certainly works, but what's the point of looping through every single model if we skip the operation for every one?

It's unlikely that these methods are called at all with disabled UUIDs, but if they are, I think they should return as quickly as possible (which is why I suggested to change the `each()` method). Alternatively we could also return early or (even better) throw an exception at the top of `generate()` and `populate()`.

Reply:
What's the point of calling `Uuids::generate()` when you have disabled UUIDs? I think we just need to make sure that the code does not break, which this does, and can focus on keeping the code concise here.

Reply:
I feel like every method should do the best it can to handle the disabled UUIDs.

One case where these methods could be called is from plugins like Janitor. They could check for `Uuids::enabled()`, but we cannot expect that everyone optimizes their code perfectly. We just cannot know when these methods are called for which reasons from which code.

We have the opportunity to do it properly, and TBH I strongly think that making code concise is really far down the priority list here. It's a good goal, but not the most important one compared to performance and energy use.

Reply:
It's not like the current solution wouldn't work 100% fine... let's get a third opinion here.

Reply:
I think we also need to consider the "debuggability" of the code. If you call the methods and they do absolutely nothing, it could take ages to find out that the reason is the disabled option. So I's much prefer exceptions.

Reply:
I agree with @lukasbestle here. I generally would always avoid any form of code execution as early as possible if it is not needed.

Reply:
@bastianallgeier What about throwing an exception instead of just returning? I think it would be much more explicit and easier to debug.

Reply:
Yep, that's much better and it's also easier to test.